### ISSUE TITLE
Consolidate Vendors init dunder function

### ISSUE THREAD

**Issue by kirilzilla** [amosproj/amos2022ws01-firmware-scraper#128]
## User story
1. As a developer
2. I want to consolidate Vendors init dunder function
3. In order to remove the redundancy and inconsistancy in the code

## Acceptance criteria
* whenever possible - consistent chrome driver settings across all vendors
* elimination of redundant, repetitive code across vendor's init dunder function

## Definition of done (DoD)
* All the acceptance criteria have been met
* The code has been merged with the main one
* The documentation and Bill of Materials have been updated
* The code is complete and uploaded in GitHub
* Functionality has been reviewed

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by IceFlovver** [comment]
class DDWRTScraper(Scraper):
def init(
self,
logger,
scrape_entry_url: str = DOWNLOAD_URL,
headless: bool = True,
max_products: int = float("inf"),
):
self.logger = logger
self.scrape_entry_url = scrape_entry_url
self.headless = headless
self.max_products = max_products
self.name = "DDWRT"

    chrome_options = Options()
    if self.headless:
        chrome_options.add_argument("--window-size=1920,1080")
        chrome_options.add_argument("--start-maximized")
        chrome_options.add_argument("--headless")
    self.driver = webdriver.Chrome(service=ChromeService(ChromeDriverManager().install()), options=chrome_options)
    self.driver.implicitly_wait(0.5)  # has to be set only once

**Comment by kirilzilla** [comment]
currently the init function in each vendor has a lot of duplicated code and inconsistent naming conventions (chromeOptions, chromeConfig, options, etc). 
we want to refactor the vendors init functions so that we get:
- cleaner code
- more robust code
- consistent chrome driver settings across all vendors (if possible)
- on par with docker / without docker
- easier feature implementation (for example argument "download_path")

**Comment by kirilzilla** [comment]
currently consolidating chrome configs for every vendor + confirming docker functionality. 
hard because some developers had to set different settings for their dev environments, which might affect docker

**Comment by kirilzilla** [comment]
chrome options for docker and python 

chrome_options.add_argument("--window-size=1920,1080")
chrome_options.add_argument("--start-maximized")
chrome_options.add_argument("--headless")
chrome_options.add_argument("--no-sandbox")
chrome_options.add_argument("--disable-dev-shm-usage")
